onnxruntime_customops/_ocos.py

```diff
@@ -65,6 +65,19 @@ def create(func, *args, **kwargs):
         add_custom_op(opdef._nativedef)
         return opdef
 
+    @classmethod
+    def get_opdef(cls, op_type):
```

Comment:
If there are many operators and if the method is called many times, it should be optimized with a dictionary.

Reply:
not used now, removed them

Reply:
not used now, removed.